app/models/link.rb

```diff
@@ -0,0 +1,27 @@
+class Link < ActiveRecord::Base
+  belongs_to :link_set
+
+  include Replaceable
+  include DefaultAttributes
+
+  validate :link_is_valid
+
+private
+  def link_is_valid
+    # Test that:
+    # - the `link_type` attribute is valid
+    # - the `target_content_id` attribute is a UUID
```

Comment:
Maybe splitting up this method in `validate :link_type_is_valid` and `validate  :content_id_is_valid` would make sense.